lib/uplink/project.go

```diff
@@ -233,3 +247,23 @@ func (p *Project) checkBucketAttribution(ctx context.Context, bucketName string)
 
 	return p.metainfo.SetAttribution(ctx, bucketName, *partnerID)
 }
+
```

Comment:
**Rational for code Changes:**
This function sets the relevant partner ID information in the protobuf message before sending it to satellite endpoint.